ironic-inspector-scripts/runhealthcheck

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
I think we got rid of these custom healthchecks already, let's not carry them over here.

Reply:
sounds good to me! :)